lib/generators/frontier_model/templates/model.rb

```diff
@@ -3,18 +3,25 @@ class <%= model_configuration.as_constant %> < ActiveRecord::Base
   # Soft delete - uses deleted_at field
   acts_as_paranoid
 
+<% # The whitespace here is very important, we only want to include a blank line if there are some enums, etc -%>
```

Comment:
yeah  - the whitespace in these templates is heinous - I wish it was smarter but it is dumb as a bag of dicks.